codec-stomp/src/test/java/io/netty/handler/codec/stomp/StompTestConstants.java

```diff
@@ -18,8 +18,8 @@
 public final class StompTestConstants {
     public static final String CONNECT_FRAME =
         "CONNECT\n" +
-            "host:stomp.github.org\n" +
             "accept-version:1.1,1.2\n" +
+            "host:stomp.github.org\n" +
```

Comment:
@Scottmitch nope there is no ordering defined in the spec, except for multiple headers with the same name. The new implementation just happens to order them this way and since the test compares for equals ...